### ISSUE TITLE
Solved

### ISSUE THREAD

**Issue by muses0229** [PKU-YuanGroup/Envision3D#4]

**Comment by Pang-Yatian** [comment]
Hi, I think your network has a problem. Please use a proxy if you are in mainland China.

**Comment by yejr0229** [comment]
Thanks for replying,and now I having trouble exporting mesh.I run this command:
python launch.py --config configs/neuralangelo-pinhole-wmask-opt.yaml --gpu 0 --train dataset.scene='a Chinese style resident building'

and I get:
Global seed set to 42
Using finite difference to compute gradients with eps=progressive
/home/omniteam/miniconda3/envs/envision3d/lib/python3.10/site-packages/lightning_fabric/connector.py:562: UserWarning: 16 is supported for historical reasons but its usage is discouraged. Please set your precision to 16-mixed instead!
  rank_zero_warn(
Using 16bit Automatic Mixed Precision (AMP)
GPU available: True (cuda), used: True
TPU available: False, using: 0 TPU cores
IPU available: False, using: 0 IPUs
HPU available: False, using: 0 HPUs
`Trainer(limit_train_batches=1.0)` was configured so 100% of the batches per epoch will be used..
[rank: 0] Global seed set to 42
Initializing distributed: GLOBAL_RANK: 0, MEMBER: 1/1
----------------------------------------------------------------------------------------------------
distributed_backend=nccl
All distributed processes registered. Starting with 1 processes
----------------------------------------------------------------------------------------------------

You are using a CUDA device ('NVIDIA A100-SXM4-80GB') that has Tensor Cores. To properly utilize them, you should set `torch.set_float32_matmul_precision('medium' | 'high')` which will trade-off precision for performance. For more details, read https://pytorch.org/docs/stable/generated/torch.set_float32_matmul_precision.html#torch.set_float32_matmul_precision
Segmentation fault (core dumped)

It's really wired and I can't locate the error, how to deal with it?
